docs/user_guide/01_Reading_data.md

```diff
@@ -1,30 +1,42 @@
 # Reading in Data
 
-The most time-consuming part of many open-source projects is getting the data in and out. This is because there are so many formats and ways a user might interact with the package. DeepForest has collated many use cases into a single `read_file` function that will attempt to read many common data formats, both projected and unprojected, and create a dataframe ready for DeepForest functions.
+## The DeepForest data model
 
-You can also optionally provide:
-  - `image_path`: A single image path to assign to all annotations in the input. This is useful when the input contains annotations for only one image.
-  - `label`: A single label to apply to all rows. This is helpful when all annotations share the same label (e.g., "Tree").
+The DeepForest data model has three components
+
+1. Annotations are stored as dataframes. Each row is an annotation with a single geometry and label. Each annotation dataframe must contain a 'image_path', which is the relative, not full path to the image, and a 'label' column.
+2. Annotation geometry is stored as a shapely object, allowing the easy movement among Point, Polygon and Box representations.
+3. Annotations are expressed in image coordinates, not geographic coordinates. There are utilities to convert geospatial data (.shp, .gpkg) to DeepForest data formats.
+
+## The read_file function
+DeepForest has collated many use cases into a single `read_file` function that will read many common data formats, both projected and unprojected, and create a dataframe ready for DeepForest functions that fits the DeepForest data model.
+
+### Example 1: A csv file containing box annotations.
 
-Example:
 ```
 from deepforest import utilities
 
-df = utilities.read_file("annotations.csv", image_path="OSBS_029.tif", label="Tree")
+df = utilities.read_file("annotations.csv", image_path="<full path to the image>", label="Tree")
 ```
 
-**Note:** If your input file contains multiple image filenames and you do not provide the `image_path` argument, a warning may appear:
+For files that lack an `image_path` or `label` column, pass the `image_path` or `label` argument.
 
+```python
+from deepforest import utilities
+
+gdf = utilities.read_file(
+    input="/path/to/annotations.shp",
+    image_path="/path/to/OSBS_029.tif",   # required if no image_path column
+    label="Tree"                        # optional: used if no 'label' column in the shapefile
+)
```

Comment:
Instead of the comments here I'd probably explain what the arguments due in the text above, which would go along with the clarification of single image/label. E.g.,

> In cases where there is a single image file, the optional `image_path` argument can be used in place of providing an `image_path` column to specify the image file location. Likewise, if there is only a single label, the optional `label` argument can be used in place of providing a `label` column to indicate the single label for all objects.